Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/MRTK.WSU.asmdef

```diff
@@ -5,7 +5,11 @@
         "Microsoft.MixedReality.Toolkit.Async",
         "Microsoft.MixedReality.Toolkit.Providers.WindowsMixedReality.Shared",
         "Microsoft.MixedReality.SceneUnderstanding.Projections.Editor",
-        "Microsoft.MixedReality.SceneUnderstanding.Projections.WSA"
+        "Microsoft.MixedReality.SceneUnderstanding.Projections.WSA",
+        "Microsoft.MixedReality.OpenXR",
+        "Microsoft.MixedReality.Toolkit.Providers.XRSDK",
+        "Unity.XR.Management",
```

Comment:
Do we need `XR.Management` here? I thought the `LoaderHelpers` should prevent that need.

Reply:
It seems that we do. After removing that line I got the following when trying to build:
Assets\MRTK\Providers\Experimental\WindowsSceneUnderstanding\WindowsSceneUnderstandingObserver.cs(134,50): error CS0012: The type 'XRLoader' is defined in an assembly that is not referenced. You must add a reference to assembly 'Unity.XR.Management, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'.